text_generation/causal_lm/cpp/causal_lm.cpp

```diff
@@ -39,43 +39,35 @@ int main(int argc, char* argv[]) try {
         }},
         {1, ov::PartialShape{
             BATCH_SIZE, -1
-        }},
-        {2, ov::PartialShape{
-	    BATCH_SIZE, -1
```

Comment:
> llama does not require position ids as input, only chatglm required

Converting it with optimum-cli as the README suggests, does generate position_ids

Reply:
Thanks for your info. We get llama OV model w/o position id with https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/convert.py
I suppose it is Optimum-Intel version issue. Let me checkout locally first.

Reply:
I think new optimum version add position_id as model input for onnx export: https://github.com/huggingface/optimum/pull/1381
@eaidova, could you add some comments for optimum-intel openvino export path? Or you recommend to use https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/convert.py for model conversion?

Reply:
Revert to use position ids for new optimum used in CI test